hardhat.config.js

```diff
@@ -197,6 +206,11 @@ module.exports = {
             sepolia: `${process.env.ETHERSCAN_API_KEY}`,
             mainnet: `${process.env.ETHERSCAN_API_KEY}`,
         },
+        zkEVMServices: {
```

Comment:
This one is not yet working, right? Maybe we should comment drop a comment that it is not yet working?

Reply:
7_claimChainInfo (that uses `zkEVMServices` in the hardhat config) is working, at least when the infrastructure is working